_api-reference/cluster-api/cluster-stats.md

```diff
@@ -30,10 +32,57 @@ All parameters are optional.
 Parameter | Type | Description
 :--- | :--- | :---
 &lt;node-filters&gt; | List | A comma-separated list of [node filters]({{site.url}}{{site.baseurl}}/api-reference/nodes-apis/index/#node-filters) that OpenSearch uses to filter results.
+metric | String | A comma-separated list of [metric groups](#metric-groups), for example, `jvm,fs`. Default is all metric groups.
+index_metric | String | A comma-separated list of [index metric groups](#index-metric-groups), for example, `docs,store`. Default is all index metrics.
 
 
-  Although the `master` node is now called `cluster_manager` for version 2.0, we retained the `master` field for backwards compatibility. If you have a node that has either a `master` role or a `cluster_manager` role, the `count` increases for both fields by 1. To see an example node count increase, see the Response sample.
-   {: .note }
+Although the `master` terminology has been deprecated for `cluster_manager` after OpenSearch 2.0, we retained the `master` field for backwards compatibility. If you have a node that has either a `master` role or a `cluster_manager` role, the `count` increases for both fields by 1. To see an example node count increase, see the [example response](#example-response).
+{: .note }
+
+### Metric groups
+
+The following table lists all available metric groups.
+
+Metric | Description
+:--- |:----
+`indices` | Statistics about indexes in the cluster.
+`os` | Statistics about the host OS including load, and memory.
+`process` | Statistics about processes, including open file descriptors and CPU usage.
+`jvm` | Statistics about the JVM, including heap usage and threads.
+`fs` | Statistics about file system usage.
+`plugins` | Statistics about OpenSearch plugins integrated within the nodes.
+`network_types` | The Transport and HTTP networks connected to the nodes.
```

Comment:
"Statistics about the"?

Reply:
This is would be a list of networks, not statistics.

Reply:
OK, then let's say "A list of the transport and HTTP networks connected to the nodes."